input-border.scss

```diff
@@ -1,15 +1,8 @@
+@import 'bower_components/vui-colors/colors.scss';
 
-$vui-input-borderWidth: 1px !default;
-$vui-input-borderColor: #cccccc !default;
-$vui-input-borderStyle: solid !default;
-$vui-input-borderRadius: 3px !default;
+$vui-input-borderRadius: 8px !default;
 
-@mixin vui-input-border(
-	$width: $vui-input-borderWidth,
-	$color: $vui-input-borderColor,
-	$style: $vui-input-borderStyle,
-	$radius: $vui-input-borderRadius
-) {
-	border: $width $style $color;
-	border-radius: $radius;
+@mixin vui-input-border() {
```

Comment:
Is this mixin intended to be "private".  In typography I prefixed with "_" for the helper mixin, maybe I should not have.  In either case, we should probably be consistent in that regard.  Thoughts?